lib/skate/telemetry.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Skate.Telemetry do
+  @moduledoc """
+  Telemetry listeners for Skate business logic.
+  """
+  require Logger
+
+  def setup_telemetry do
+    _ =
+      :telemetry.attach(
+        "skate",
+        [:skate, :repo, :query],
+        &Skate.Telemetry.handle_event/4,
+        %{}
+      )
+  end
+
+  def handle_event([:skate, :repo, :query], measurements, metadata, _config) do
+    if metadata.source == "detours" do
+      {:ok, result} = metadata.result
+
+      Logger.info(fn ->
+        "Telemetry for Detours query, connection_id=#{result.connection_id} num_rows=#{result.num_rows} decode_time=#{measurements.decode_time} query_time=#{measurements.query_time} total_time=#{measurements.total_time} query=#{metadata.query}"
```

Comment:
I think we should quote `query` so that Splunk can grab all the contents of the `query` value using it's `key="value"` extractor, I'm not actually _certain_ that it works with single quotes though[^1]? 

```suggestion
        "Telemetry for Detours query, connection_id=#{result.connection_id} num_rows=#{result.num_rows} decode_time=#{measurements.decode_time} query_time=#{measurements.query_time} total_time=#{measurements.total_time} query='#{metadata.query}'"
```

[^1]: we'd need to use single quotes though because the query has double quotes in it.

Reply:
Ahhhh definitely! 🚀